### ISSUE TITLE
ENOENT: no such file or directory when running npm install on kitchen sink example

### ISSUE THREAD

**Issue by juanpasolano** [vercel/turborepo#425]
### What version of Turborepo are you using?

1.0.23

### What package manager are you using / does the bug impact?

npm

### What operating system are you using?

Mac

### Describe the Bug

I'm simply running `npm install` on your kitchen-sink example and it gives me this error.
```bash
npm ERR! code ENOENT
npm ERR! syscall chmod
npm ERR! path /project-path/node_modules/scripts/node_modules/.bin/jest
npm ERR! errno -2
npm ERR! enoent ENOENT: no such file or directory, chmod '/project-path/node_modules/scripts/node_modules/.bin/jest'
npm ERR! enoent This is related to npm not being able to find a file.
npm ERR! enoent 
```

The error does not occur on the basic nor the design-system example folders.

### Expected Behavior

Npm installs correctly

### To Reproduce

I cloned your repo and runned `npm install` on the kitchen sink example.
I tried this with.  
node -v `14.17.1`  npm `8.3.0`. 
And also node -v `16.13.1` npm `8.3.0`.

**Comment by jaredpalmer** [comment]
This has recently been fixed today in #432. Sorry about that!

**Comment by juanpasolano** [comment]
uh damn that was faaast! ❤️
